lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var _ = require('lodash');
```

Comment:
Proxies and firewalls can sometimes close inactive connections. To prevent them from closing a connection when using  server side events, you can send a heartbeat after some period of inactivity, keeping the connection active.